.github/workflows/generate-feature-graphic.yml

```diff
@@ -0,0 +1,60 @@
+name: Generate feature graphic
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'fastlane/**'
+jobs:
+  generate-feature-graphic:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install requirements
+      run: |
+        sudo apt-get update
+        sudo apt-get install optipng
+        # Install 200 weight versions of relevant Noto (to use for languages not supported by Lexend Deca)
+        sudo apt-get install fonts-noto-extra fonts-noto-cjk-extra
+        # Custom fonts
+        mkdir "$HOME/.fonts"
+        find .scripts/generate_feature_graphic/fonts -name '*.ttf' -exec cp {} "$HOME/.fonts" \;
+        fc-cache
+    - name: Generate featureGraphic.png for each language
+      run: |
+        for lang in fastlane/metadata/android/*; do
+          pushd "$lang"
+          # Extract text after 'Catima - '
+          subtext=$(grep -oP '(?<=Catima \S ).*' title.txt || true)
+          # Generate temporary .svg with correct text
+          sed "s/Loyalty Card Wallet/$subtext/" ../../../../.scripts/generate_feature_graphic/featureGraphic.svg > featureGraphic.svg
+          # Set correct fallback font for language if needed (Lexend Deca has limited support)
+          # We specifically need the Serif version because of the 200 weight
+          case "$(basename $lang)" in
+            bg) sed -i "s/Lexend Deca/Noto Serif/" featureGraphic.svg ;;
+            el-GR) sed -i "s/Lexend Deca/Noto Serif/" featureGraphic.svg ;;
+            ja-JP) sed -i "s/Lexend Deca/Noto Serif CJK JP/" featureGraphic.svg ;;
+            ko) sed -i "s/Lexend Deca/Noto Serif CJK KR/" featureGraphic.svg ;;
+            ro) sed -i "s/Lexend Deca/Noto Serif/" featureGraphic.svg ;;
+            ru-RU) sed -i "s/Lexend Deca/Noto Serif/" featureGraphic.svg ;;
+            uk) sed -i "s/Lexend Deca/Noto Serif/" featureGraphic.svg ;;
+            zh-CN) sed -i "s/Lexend Deca/Noto Serif CJK SC/" featureGraphic.svg ;;
+            zh-TW) sed -i "s/Lexend Deca/Noto Serif CJK TC/" featureGraphic.svg ;;
```

Comment:
```suggestion
            bg|el-GR|ro|ru-RU|uk) sed -i "s/Lexend Deca/Noto Serif/" featureGraphic.svg ;;
            ja-JP) sed -i "s/Lexend Deca/Noto Serif CJK JP/" featureGraphic.svg ;;
            ko) sed -i "s/Lexend Deca/Noto Serif CJK KR/" featureGraphic.svg ;;
            zh-CN) sed -i "s/Lexend Deca/Noto Serif CJK SC/" featureGraphic.svg ;;
            zh-TW) sed -i "s/Lexend Deca/Noto Serif CJK TC/" featureGraphic.svg ;;
```

Reply:
If you do that, might want to skip `ro` for now since it doesn't have a title. So if you do use "Loyalty Card Wallet" it'll have a different font for no reason.

Reply:
Or better, add `ar` etc. and have it only replace the font if the text is not the default.

Reply:
Maybe just remove `ro` for now. And add the other languages when you actually get an updated title to test with. Probably makes the most sense.